src/include/cart/swim.h

```diff
@@ -208,7 +208,8 @@ int swim_progress(struct swim_context *ctx, int64_t timeout);
  *                   ALL timeouts will be shifted.
  * @returns          0 on success, negative error ID otherwise
  */
-int swim_net_glitch_update(struct swim_context *ctx, uint64_t delay);
+int swim_net_glitch_update(struct swim_context *ctx, swim_id_t id,
```

Comment:
minor: id field not documented in params above

Reply:
done